amp/repository/aim/view/auditLoggerManager.jsp

```diff
@@ -56,6 +56,26 @@ function actionChanged(value){
 	}
 }
 
+function filterChanged(value){

```

Comment:
I wouldnt allow to filter only by one option, since the user might want to filter by date _and_ team or team _and_ user. besides that it makes the code more complex